horace_core/sqw/multifit_subclasses/mfclass_Horace_sqw_sqw.m

```diff
@@ -821,7 +821,7 @@
                 wrapfun.bp_wrap = append_args (wrapfun.bp_wrap, '-ave');
                 obj_tmp.wrapfun = wrapfun;
             end
-            [data_out, calcdata, ok, mess] = fit@mfclass (obj_tmp, varargin{:});
+            [data_out, calcdata] = fit@mfclass (obj_tmp, varargin{:});
```

Comment:
ok,mess are return values for this function and will now not be set. If not removed from this function, they should be set true,"" presuming that any other values will now be dealt with by an error throw in the fit call